lib/app/view/app.dart

```diff
@@ -113,6 +113,7 @@ class _AppViewState extends State<AppView> {
             Locale('pa'),
             Locale('he'),
             Locale('kn'),
+            Locale('pt', 'br'),
```

Comment:
Sorry for previous comment, didn't know Flutter ignores country code for filename. It works if we change this to
```
            Locale('pt', "BR"),
```